hooks/tk-multi-publish2/exports/base_export_hook.py

```diff
@@ -0,0 +1,35 @@
+import SyPy3
+from SyPy3.sylevel import SyLevel
+from SyPy3.syobj import SyObj
+
+def prepare(engine, settings, item):
+    """
+    Prepares SynthEyes for the imminent export. 
+    This is highly dependant on the specific purpose of each hook and the implementation and settings of the associated exporter. 
+    Drastically altering the scene file should be avoided. All changes to the SynthEyes scene should be done in Begin-Accept-blocks.
+    This is necessary to automatically revert any changes after the export.
+    Depending on whether the associated exporter supports it, the best way to exclude objects from the export process is to set their 
+    'isExported'-attribute to false prior to the export.
+    """
+    hlev: SyLevel = engine.get_syntheyes_connection()
+        
+    hlev.Begin()
+    try:
+        pass
```

Comment:
Wenn du in try nur passed sollte keine Exception geraised werden. Dann könntest du es vereinfachen

Reply:
Der try-Block in dieser Form mit hlev.Begin()/Accept() findet sich so in einigen Dateien in dem Repo wieder. Grund dafür ist, dass SynthEyes für manche Changes/Operations zwangsweise einen Undo-Block fordert - das passiert genau durch das Wrappen des Codes in hlev.Begin()/Accept(). Wenn aber jetzt Begin() gecalled wird, dann eine Exception irgendeiner Art auftritt und Accept() nie erreicht wird, hängt sich SynthEyes vollständig auf bis wieder ein Accept()-Block erreicht wird. Daher hab ich diese Blocks lieber als Sicherheit drin, selbst wenn der Code in dem Block eigentlich unproblematisch sein sollte.